README.md

```diff
@@ -1,6 +1,27 @@
-# Bespoke Curator
+<p align="center">
+  <a href="https://bespokelabs.ai/" target="_blank">
+    <img src="./docs/Bespoke-Labs-Logo-on-Mint.png" width="50%" alt="Bespoke Labs Logo" />
+  </a>
+</p>
+
+<h1 align="center">Curator</h1>
```

Comment:
Thinking out loud here for alternative layout of logo and title. 

Ideally the text font can be consistent for bespoke labs (in logo) and curator IMO, 
right now it looks like this in my preview
<img width="771" alt="image" src="https://github.com/user-attachments/assets/28995cd1-bae6-46c7-ba7f-0602c02c2f4f">

something from my gut telling me having consistent font can be more appealing and just cleaner in general. 
<img width="244" alt="image" src="https://github.com/user-attachments/assets/e0ff12ca-c3fd-4bf0-9686-8b7b11b8ffef">
Something like this is cleaner to me.

Alternative, we create a logo just for bespoke labs curator, including the text (example vllm)

Or alternatively, we don't have logo, and just put bespoke labs curator as title (example llama).

Reply:
I didn't realize the effect on dark. Changed it now so that it works for both the themes.
Light theme:
<img width="487" alt="image" src="https://github.com/user-attachments/assets/dcd7753a-d928-44f7-901c-6154242c9ab6">
Dark theme:
<img width="471" alt="image" src="https://github.com/user-attachments/assets/ae8bcde4-56bb-4dcd-8a38-088c3b5db3bc">

Also, I want to keep the Logo + Bespoke Labs at the top. 
And given we are small, we need one consistent logo, so new logo for Curator may not be a good idea.
We can definitely explore having Logo + Bespoke Labs Curator later.

Reply:
Ok added just the logo. We can make Logo + Bespoke Labs Curator in photoshop later (Alex will try to do that).

Reply:
done

Reply:
<img width="998" alt="image" src="https://github.com/user-attachments/assets/cc9492cf-4bd3-4cae-a220-81ff0110754e">
perfect